backend/src/v5/services/eventsListener/components/projectEvents.js

```diff
@@ -0,0 +1,39 @@
+/**
+ *  Copyright (C) 2022 3D Repo Ltd
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+const { EVENTS: chatEvents } = require('../../chat/chat.constants');
+const { createProjectMessage } = require('../../chat');
+const { events } = require('../../eventsManager/eventsManager.constants');
+const { logger } = require('../../../utils/logger');
+const { subscribe } = require('../../eventsManager/eventsManager');
+
+const modelAdded = async ({ teamspace, project, model, data, isFederation }) => {
```

Comment:
wrong place

This is a model event.

(even though in the chat service, it's broadcasted in the project room)

There's a difference between the event managers within the backend and the chat service events, they're 2 different things